client/src/index.ts

```diff
@@ -280,22 +282,14 @@ export class RestRepository<T> extends Repository<T> {
     super(null, table, {});
     this.client = client;
 
-    const { fetch } = client.options;
+    const doWeHaveFetch = typeof fetch !== 'undefined';
```

Comment:
What do you think if we check there's a `fetch` key in the options object and then do something if it's `undefined`

> const doWeHaveFetch = 'fetch' in client.options;
> if (typeof client.options.fetch === 'undefined') // the user wanted to pass something, but it's undefined. Wrong import/require() ?

Reply:
Sure, why not? Something like e1d5b16f26d2dfce04d2920fa44b1933d1633508?